surfsense_backend/app/routes/search_source_connectors_routes.py

```
@@ -263,48 +306,64 @@ async def delete_search_source_connector(
     try:
         # Check if the connector belongs to the user
-        connector = await check_ownership(session, SearchSourceConnector, connector_id, user)
-        
+        connector = await check_ownership(
+            session, SearchSourceConnector, connector_id, user
+        )
+
         # Check if the search space belongs to the user
-        search_space = await check_ownership(session, SearchSpace, search_space_id, user)
-        
+        search_space = await check_ownership(
```

Ruff (0.12.2)

Local variable `search_space` is assigned to but never used

Remove assignment to unused variable `search_space`

(F841)